src/ddmd/expression.d

```diff
@@ -15410,6 +15411,25 @@ extern (C++) final class CmpExp : BinExp
             if ((t1.ty == Tarray || t1.ty == Tsarray) && (t2.ty == Tarray || t2.ty == Tsarray))
             {
                 semanticTypeInfo(sc, t1.nextOf());
+
+                // Lowering to template call
+                Identifier id = Identifier.idPool("__cmp");
```

Comment:
call it __cmp or something like that since that is the id it represents.
Also maybe it would be useful to add it to `idgen.d`
Such that you don't have to call idPool every time, which is not a cheap call.